mlrun/__main__.py

```diff
@@ -145,7 +146,8 @@ def run(
     hyperparam,
     param_file,
     selector,
-    tuning_strategy,
+    strategy,
+    hyper_options,
```

Comment:
I see this is repeating everywhere, just like that class is `HyperParamOptions` I don't see a reason why it won't be `hyper_param_options` everywhere

Reply:
too long IMO

Reply:
done